ui/app/templates/application.hbs

```diff
@@ -10,6 +10,11 @@
         <p data-test-error-message class="subtitle">
           The cluster has no leader. <a href="https://www.nomadproject.io/guides/outage.html"> Read about Outage Recovery.</a>
         </p>
+      {{else if this.isOTTExchange}}
+        <h1 data-test-error-title class="title is-spaced">Token Exchange Error</h1>
+        <p data-test-error-message class="subtitle">
+          Failed to exchange the one-time token.
+        </p>
```

Comment:
I followed the existing `no-leader-error` pattern and created the `ott-exchange-error` that’s checked for here to display this custom message… maybe a more generic known error type with custom title and message would be useful eventually 🤔

Reply:
Yeah. Errors and notifications I think are high on the "to refactor" list.